src/main/java/dukebot/Duke.java

```diff
@@ -0,0 +1,107 @@
+package dukebot;
+
+import dukebot.command.Command;
+import dukebot.command.Parser;
+import dukebot.exception.DukeException;
+import dukebot.storage.Storage;
+import dukebot.tasklist.Task;
+import dukebot.tasklist.TaskList;
+import dukebot.ui.Ui;
```

Comment:
Good packaging!

Reply:
Good use of OOP! Perhaps you could elaborate more in your javadoc on the different enums used for the Ui?